.github/scripts/generate_release_notes.py

```diff
@@ -108,22 +109,27 @@ def get_repo_priority(name):
 for c in cards:
     issueOrPR = c.get_content()
     contributors = []
-    url = ""
+    url = issueOrPR.html_url
     try:
         # only a PR can be converted to a PR object, otherwise will throw error.
         pr = issueOrPR.as_pull_request()
         contributors.append(pr.user.login)
-        url = pr.html_url
     except:
         a = [l.login for l in issueOrPR.assignees]
         contributors.extend(a)
```

Comment:
check if `a` is not empty, if so log that it is empty?

Reply:
it is not a bug, but will help us track unassigned issues.

Reply:
ok.